web/src/show.php

```diff
@@ -84,7 +84,8 @@
     <ul class="dupes">
         <?php foreach ($similarStudents as $index => $record): ?>
             <li>
-                <a href="?id=<?php out($record['other_id']) ?>"><?php out(chr(66 + $index)) ?></a>: <?php out(sprintf('%.0f%%', 100 * $record['weighted_mean'])) ?>
+                <a class=<?php out(chr(82 + $index))?> href="?id=<?php out($record['other_id']) ?>"><?php out(chr(66 + $index)) ?></a>: <?php out(sprintf('%.0f%%', 100 * $record['weighted_mean'])) ?>
```

Comment:
Why 82? That means you start at "R", but can only get 9 different class names before reaching Z, while the colours support 12 steps, so – including the original record – you would need to cater for 13 different classes.